modules/core/keeper/keeper.go

```diff
@@ -60,13 +60,13 @@ func NewKeeper(
 	}
 
 	clientKeeper := clientkeeper.NewKeeper(cdc, key, paramSpace, stakingKeeper, upgradeKeeper)
```

Comment:
Ideally, I think we should be replacing the `stakingKeeper` arg to core ibc's `NewKeeper` with a `types.ConsensusHost` and have users pipe in whatever consensus host they are using explicitly in app.go. 

E.g. `ibctm.NewConsensusHost(app.StakingKeeper)` in app.go as an arg to `ibc.NewKeeper(...)`

thoughts?